sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleFacadeEjb.java

```diff
@@ -423,13 +412,24 @@ public SampleReferenceDto getReferenceByUuid(String uuid) {
 				sample.get(Sample.ADDITIONAL_TESTING_REQUESTED),
 				cb.isNotEmpty(sample.get(Sample.ADDITIONAL_TESTS)),
 				districtSelect,
-				joins.getReportingUser().get(User.UUID),
-				joins.getLab().get(Facility.UUID)));
-		selections.addAll(getCaseJurisdictionSelections(joins));
-		selections.addAll(getContactJurisdictionSelections(joins));
-		selections.addAll(getEventJurisdictionSelections(joins));
-
-		cq.multiselect(selections);
+				joins.getLab().get(Facility.UUID),
+			JurisdictionHelper.jurisdictionSelector(cb, sampleService.inJurisdictionOrOwned(cb, joins)),
+			JurisdictionHelper.jurisdictionSelector(
+				cb,
+				cb.and(cb.isNotNull(joins.getCaze()), caseService.inJurisdictionOrOwned(cb, new CaseJoins<>(joins.getCaze())))),
```

Comment:
are the `isNotNull` conditions needed, I didn't see them in other places

Reply:
without them depending on the state of the entity and the associated objects JPA will throw an error